torch/fx/experimental/symbolic_shapes.py

```diff
@@ -127,7 +128,7 @@ def clone(self):
         return PySymInt(self.expr, self.shape_env, constant=self.constant)
 
     def _update_expr(self):
-        self._expr = self.shape_env.simplify(self._expr)
+        self._expr = self.shape_env.replace(self._expr)
```

Comment:
Really?

Reply:
Mmm... yeah. Otherwise sympy "helpfully" does simplifications like

```
(x + 1)/5 => x/5 + 1/5
```

Reply:
But isn't that reverting what the PR just below in the stack is doing partially?

Reply:
Yeah. Ideally I would just modify it in the PR below, but don't wanna restart CI 🤔